README.md

```diff
@@ -1,2 +1,153 @@
 # ilib-loctool-mrkdwn
-A loctool plugin that can parse Slack mrkdwn messages in a json file
+
+Ilib loctool plugin to parse and localize the slack-flavored markdown
```

Comment:
What is `mrkdwn` version which is supported by this tool?

Reply:
Apparently, Slack does not version its mrkdwn specs: https://api.slack.com/reference/surfaces/formatting